Pinta/Actions/File/SaveDocumentImplementationAction.cs

```diff
@@ -118,9 +118,11 @@ private bool SaveFileAs (Document document)
 		// If we already have a format, set it to the default.
 		// If not, default to jpeg
 		FormatDescriptor? format_desc = null;
+		FormatDescriptor? previous_format_desc = null;
 
 		if (document.HasFile)
-			format_desc = image_formats.GetFormatByFile (document.DisplayName);
+			previous_format_desc = image_formats.GetFormatByFile (document.DisplayName);
+			format_desc = previous_format_desc;
```

Comment:
Should there be braces around these two statements? I think both are supposed to be in the `if` block

Reply:
right, i'm really used to python and the lack of brackets made me forgot i needed to add them :sweat_smile: